docs/play/cdc-pay-integration.md

```diff
@@ -0,0 +1,194 @@
+# CDC Pay Integration
+This page illustrates some integration options for the CDC Pay module. The CDC Pay module can be intergated directly into your game development environment.
+
+## Pre-requisites
+
+- Crypto.com Pay Developer [access](https://merchant.crypto.com/users/sign_in/)
+- Optional:
+  - Webhook server (e.g. nodejs, [ngrok](https://ngrok.com/)) 
+
+### Overview
+The CDC Pay module makes use of REST API calls to trigger certain actions. These action will create events that are retrievable via webhooks. The following shell command illsutrates how to initiate the CDC Pay payment.
+
+```sh
+POST https://pay.crypto.com/api/payments
+```
+```sh
+curl https://pay.crypto.com/api/payments \
+  -u SECRET_KEY: \
+  -d amount=2500 \
+  -d currency=USD \
+  -d description="Crypto.com Tee (Unisex)"
+```
+
+::: tip
+This API can be called using either your Secret Key or Publishable Key. These keys are generated in the merchant center [dashboard](https://merchant.crypto.com/users/sign_in/).
+:::
+
+## Integrating with Unity
+To use the CDC Pay module in Unity we can simple use the [UnityWebRequest](https://docs.unity3d.com/ScriptReference/Networking.UnityWebRequest.Post.html) module to simulate a post request. We recommend to use a webhook server to retrieve the payment event responses.
+
+- Create payment example
+    ```csharp
+    using UnityEngine;
+    using UnityEngine.Networking;
+    using System.Collections;
+
+    public class CDCPayment : MonoBehaviour
+    {
+        void Start()
+        {
+            StartCoroutine(Pay());
+        }
+
+        IEnumerator Pay()
+        {
+            WWWForm form = new WWWForm();
+            form.AddField("amount", "1000");
+            form.AddField("currency", "USD");
+            form.AddField("description", "Product Name");
+            form.AddField("return_url", "http://YOUR_WEBSITE.com/orders/123/complete");
+            form.AddField("cancel_url", "http://YOUR_WEBSITE.com/orders/123/fail");
+
+            using (UnityWebRequest www = UnityWebRequest.Post("https://pay.crypto.com/api/payments", form))
+            www.SetRequestHeader("Authorization", "Bearer SECURITY_KEY");
```

Comment:
I guess good to highlight this should be the "publishable key", not the "secret key", as it's done in the client

Reply:
fixed